README.md

```diff
@@ -128,9 +128,28 @@ require 'queue_classic'
 FailedQueue = QC::Queue.new("failed_jobs")
 
 class MyWorker < QC::Worker
+
+  # A job contain these attributes:
+  # :id Integer, the job id
+  # :method String, containing the object and method
+  # :args String, the arguments
+  # :q_name String, the queue name
+  # :scheduled_at Time, the scheduled time if the job was scheduled
+
+  # Execute the job using the methods and arguments
+  def call(job)
```

Comment:
It's a part of the previous PR but translated for the job-is-a-hash, I just added a comment to tell explicitely that job is a Hash